src/style.css

```diff
@@ -1 +1,10 @@
 /* styles go here */
+.form__field {
+  display: block;
+}
+input {
+  margin-bottom: 10px;
+}
+fieldset {
+margin-bottom: 20px;
+}
```

Comment:
```suggestion
}

input {
  margin-bottom: 10px;
}

fieldset {
margin-bottom: 20px;
}
```

Reply:
I understood my mistakes, but I would like to know more precisely if I need to fix it myself in vsc and then do "push" again, or do "commit suggestion" -> "commit changes". Thank you in advance!